packages/entities/entities-redis-configurations/src/components/RedisConfigurationList.vue

```diff
@@ -0,0 +1,420 @@
+<template>
+  <div class="kong-ui-entities-partials-list">
+    <EntityBaseTable
+      :cache-identifier="cacheIdentifier"
+      :disable-sorting="disableSorting"
+      :empty-state-options="emptyStateOptions"
+      enable-entity-actions
+      :error-message="errorMessage"
+      :fetcher="fetcher"
+      :fetcher-cache-key="fetcherCacheKey"
+      pagination-type="offset"
+      preferences-storage-key="kong-ui-entities-redis-configuration-list"
+      :query="filterQuery"
+      :table-headers="tableHeaders"
+      @clear-search-input="clearFilter"
+      @click:row="(row: any) => rowClick(row as EntityRow)"
+      @sort="refreshList"
+    >
+      <!-- Filter -->
+      <template #toolbar-filter>
+        <EntityFilter
+          v-model="filterQuery"
+          :config="filterConfig"
+        />
+      </template>
+      <!-- Create action -->
+      <template #toolbar-button>
+        <PermissionsWrapper :auth-function="() => canCreate()">
+          <KButton
+            appearance="primary"
+            data-testid="toolbar-add-redis-configuration"
+            size="large"
+            :to="config.createRoute"
+          >
+            <AddIcon />
+            {{ t('actions.create') }}
+          </KButton>
+        </PermissionsWrapper>
+      </template>
+
+      <!-- Column Formatting -->
+      <template #name="{ rowValue }">
+        <b>{{ rowValue ?? '-' }}</b>
+      </template>
+      <template #type="{ row }">
+        {{ renderRedisType(row) }}
+      </template>
+      <template #plugins="{ row }">
+        <LinkedPluginsInline
+          :config="config"
+          :partial-id="row.id"
+          @click.stop="showLinkedPlugins(row.id)"
+        />
+      </template>
+
+      <!-- Row actions -->
+      <template #actions="{ row }">
+        <PermissionsWrapper :auth-function="() => canRetrieve(row)">
+          <KDropdownItem
+            data-testid="action-entity-view"
+            has-divider
```

Comment:
This adds a line above the item:
<img width="614" alt="image" src="https://github.com/user-attachments/assets/41d6f57c-c14a-4424-ac25-afd47b834369" />

I think we can either remove this prop or add a `Copy ID` item, just like what other entity lists have.

Reply:
added